templates/common/_base/units/machine-config-daemon-host.service.yaml

```diff
@@ -18,8 +18,6 @@ contents: |
   # TODO add --from-etc-pullspec after ratcheting
   ExecStart=/usr/libexec/machine-config-daemon pivot
 
-  [Install]
-  WantedBy=multi-user.target
```

Comment:
+1 to remove this as it may not adding up in this service.

Speaking out loud:
From the documentation https://www.freedesktop.org/software/systemd/man/systemd.unit.html ,  it says `WantedBy=  option may be used more than once, or a space-separated list of unit names may be given. A symbolic link is created in the .wants/ or .requires/ directory of each of the listed units when this unit is installed by systemctl enable.` By default m-c-d-host.service is in disabled state, so I hope this wasn't causing any issue so far.